graph_nets/tests/blocks_test.py

```diff
@@ -646,6 +678,36 @@ def test_output_values(
     self.assertNDArrayNear(
         expected_output_nodes, output_graph_out.nodes, err=1e-4)
 
+  @parameterized.named_parameters(
+      ("only scaling", 2, 0),
+      ("only offsetting", 0, 2),
+      ("scaling and offsetting", 2, 2),
+      ("without scaling and offsetting", 1, 0),
+  )
+  def test_optional_arguments(self, scale, offset):
```

Comment:
A slightly simpler way to test this, that does not involve forking all of the concat code, etc is to compare the outputs of two identical models, one that uses a model_fn like:

```
fixed_scale = 10
fixed_offset = 5
model_fn = lambda: lambda features: features * fixed_scale + fixed_offset
```

and another one that uses:
`model_args_fn = lambda: lambda features, scale, offset: features * scale + offset`

And simply compare that the outputs are the same, rather than building expected values form scratch.